backend/app/assets/javascripts/spree/backend/calculator.js

```diff
@@ -14,6 +17,14 @@ Spree.CalculatorEditView = Backbone.View.extend({
       $(this).find(':input').prop("disabled", !selected);
       $(this).toggle(selected);
     });
+  },
+
+  // Somehow backbone seems to prevent events from bubbling up the DOM
+  submit: function(e) {
+    var code = e.keyCode || e.which;
+    if (code === 13) {
+      this.$el.closest('form').submit();
+    }
```

Comment:
I'd rather find a different solution to this. I believe I've found the issue and we should address it in a separate PR. It's not a backbone issue.

The is that the "Add" buttons from the Tiered calculators, which aren't submit buttons, but their disabled state causes chrome to not submit the form on "Enter". This can probably be better solved by making these buttons an `<a class="btn"` instead.

Reply:
Great. I’m more than happy to remove this

Reply:
Converted the `button`s into `a`nchors. The actual problem was that hitting the enter key added new tiers and that prevents the form from being submitted.